electron/js/preload.js

```
@@ -137,6 +137,11 @@ ipcRenderer.once('webapp-loaded', function(sender, config) {
     });
   }
   // else we are on /auth
+  try {
+    Object.assign(window.sodium, require('libsodium-neon'));
+  } catch (err) {
+    // fallback to libsodium.js
```

We can, but I thought the idea was to silently fail. :)